osu.Game/Rulesets/RulesetSelector.cs

```diff
@@ -13,11 +14,27 @@ public abstract class RulesetSelector : TabControl<RulesetInfo>
 
         protected override Dropdown<RulesetInfo> CreateDropdown() => null;
 
+        protected virtual bool SelectInitialRuleset => true;
+
+        protected RulesetSelector()
+        {
+            SelectFirstTabByDefault = false;
+        }
+
         [BackgroundDependencyLoader]
         private void load()
         {
             foreach (var r in Rulesets.AvailableRulesets)
                 AddItem(r);
+
+            if (SelectInitialRuleset)
```

Comment:
i don't even know what this is.

Reply:
I don't think I'm gonna discuss about this one, relying on the ruleset selector is a whole issue of its own that I'm gonna rewrite.